.golangci.yml

```diff
@@ -15,6 +16,8 @@ linters:
     - errcheck
     - scopelint
     - varcheck
+    - funlen
```

Comment:
Remove; We shouldn't disable this. Any new functions should ideally be refactored and existing functions at the very least include `// nolint: funlen`.